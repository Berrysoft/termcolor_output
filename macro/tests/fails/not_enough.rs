use termcolor_output as tco;

fn main() {
    let writer: Vec<u8> = vec![];
    tco::write!(
        writer,
        "Before first - {}, then - {:?}, or even {{this}}: {:b}!"
    );
}
