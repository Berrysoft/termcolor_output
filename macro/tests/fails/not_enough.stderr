error: Not enough input parameters for this format string
 --> $DIR/not_enough.rs:7:9
  |
7 |         "Before first - {}, then - {:?}, or even {{this}}: {:b}!"
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
